packages/qwik/src/optimizer/src/manifest.ts

```diff
@@ -270,7 +274,13 @@ export function generateManifestFromBundles(
     return canonPath(bundle.fileName);
   };
   // We need to find our QRL exports
-  const qrlNames = new Set([...segments.map((h) => h.name)]);
+  const qrlNames = new Set(segments.map((h) => h.name));
+  for (const symbol of extraSymbols) {
+    qrlNames.add(symbol);
+  }
+  const taskNames = new Set(
+    segments.filter((h) => /use[a-zA-Z]*Task(_\d+)?$/.test(h.displayName)).map((h) => h.name)
```

Comment:
maybe we can create regexp once and reuse it? not sure how about performance here

Reply:
it gets optimized, I tested it a while back